### PR TITLE
Issue #10. HTML markup for header region(excluding menu).

### PR DESCRIPTION
**Pull request by nhiieL**

### CODE REVIEW

File: static/sass/partials/_form.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    .input-text {
+      font-size: $base-font-size;
+      border: 1px solid $secondary-color;
+      padding: 6px;
+      width: 95%;
+    }
+    .submit-btn {
+      font-size: $base-font-size;
+      background: #b47b31;
```
Comment (marychris) [inline static/sass/partials/_form.scss]: Use variables for colors. ' #b47b31'

File: static/sass/partials/_form.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      color: $primary-bg;
+      border: none;
+      padding: 7px;
+      width: 100%;
+    }
+    .submit-btn:hover {
+      background-color: #b07337;
```
Comment (marychris) [inline static/sass/partials/_form.scss]: Use variables for '#b07337', if variable does not exist, create one. You can call this variable $hover.

File: static/sass/partials/_form.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      color: #000;
```
Comment (marychris) [inline static/sass/partials/_form.scss]: Use variable for #000;

File: static/index.html [hunk 4]
```diff
@@ -20,12 +20,29 @@
      <header>
        <div class="container">
            <div class="row">
-         </div>
-       </div>
+               <div class="col-sm-6">
+                   <div class="logo">
+                     <img src="images/Official_logo.jpg">
```
Comment (Luukyb) [inline static/index.html]: As a note, we will need to replace this later with a SVG I think. It would be lighter and better quality.

**Pull request merged**